CHANGELOG.md

```diff
@@ -7,11 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [rust-0.4.1] - 2023-10-16
```

Comment:
You need to add the entry at the bottom of this file to match the `rust-0.4.1` version